__tests__/app.test.js

```diff
@@ -62,6 +62,7 @@ describe("GET articles by ID", () => {
           votes: 100,
           article_img_url:
             "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+            comment_count: 11
```

Comment:
As a rule we wouldn't want to be going back and changing tests, we have used to match object here so this wont break if we added another property to the return value. We should test for new added value separately as an extra test